c_gen/c_test_gen.py

```diff
@@ -1913,7 +1913,7 @@ def gen_ident_tests(out):
 
 """)
 
-    for ident, info in of_g.identifiers.items():
+    for ident, info in sorted(of_g.identifiers.items()):
```

Comment:
Could possibly change `c_gen/of_g_legacy.py#L240` to OrderedDict?

Reply:
yes, changed to OrderedDict